build/build.proj

```diff
@@ -108,7 +108,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildNuGetPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildNuGetPackages;BuildModernVsixPackages;Test" />
```

Comment:
@davkean Let me forward you the thread on which we were asked to separate out building V3 VSIX into a separate command. We should follow up with the microbuild team if this is now fixed.